src/wallet/wallet.cpp

```diff
@@ -2156,9 +2130,26 @@ void CWallet::ResendWalletTransactions(interfaces::Chain::Lock& locked_chain, in
 
     // Rebroadcast unconfirmed txes older than 5 minutes before the last
     // block was found:
-    std::vector<uint256> relayed = ResendWalletTransactionsBefore(locked_chain, nBestBlockTime-5*60);
-    if (!relayed.empty())
+    std::vector<uint256> relayed;
+
+    { // cs_wallet scope
+    LOCK(cs_wallet);
+
+    // Relay transactions
+    for (std::pair<const uint256, CWalletTx>& item : mapWallet)
```

Comment:
```suggestion
    for (auto& item : mapWallet)
```

Style-nit: Since this isn't move only of the above method anyway, you are welcome to do style fixups

Reply:
ok, fixed this and some other style nits.